GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -372,18 +203,8 @@ public void export(Map<String, String> criteria, File fileName,
         try {
             Map<QuestionGroupDto, List<QuestionDto>> questionMap = 
                     loadAllQuestions(surveyId, performGeoRollup, serverBaseUrl, apiKey);
-            //minimal data plus cascade level names
-            if (useQuestionId && DEFAULT_LOCALE.equals(locale)) {
-                loadQuestionOptions(surveyId, serverBaseUrl, questionMap, apiKey);
```

Comment:
Where will the question options be loaded from?

Reply:
They don't need translated names anymore.